db/migrate/20190802133303_create_triggers_backend_apis.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class CreateTriggersBackendApis < ActiveRecord::Migration
+  # At this point it does not matter if it blocks these tables or not because they are not being used yet anyway
+
+  def up
+    self.class.execute_trigger_actions do |trigger|
+      connection = ActiveRecord::Base.connection
+      connection.execute(trigger.drop)
+      connection.execute(trigger.create)
+    end
+  end
+
+  def down
+    self.class.execute_trigger_actions do |trigger|
+      ActiveRecord::Base.connection.execute(trigger.drop)
+    end
+  end
+
+  def self.execute_trigger_actions
+    BackendApi.transaction do
+      System::Database.triggers.each do |trigger|
```

Comment:
It could also be

```ruby
triggers = System::Database.triggers.select { |trigger| trigger.table =~ /\Abackend_api((s)|(_configs))\z/ }
triggers.each(&block)
```

IDK if it helps in terms of readability.